plugin/trino-bigquery/src/test/resources/import-tpch-to-bigquery.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -euo pipefail
+cd "${BASH_SOURCE%/*}"
+
+# TODO: Adjust these values as needed
+trino_cli="<PATH_To_TRINO_CLI>"
```

Comment:
Use `client/trino-cli/target/trino-cli-*-executable.jar` automatically.

If it doesn't exist, fail asking user to fully build the project.